Tests/SwiftDocCTests/Infrastructure/DocumentationBundleInfoTests.swift

```diff
@@ -330,4 +330,53 @@ class DocumentationBundleInfoTests: XCTestCase {
             XCTAssertEqual(error.localizedDescription, "Unable to decode Info.plist file. Verify that it is correctly formed. Value missing for key inside <dict> at line 24")
         }
     }
+    
+    func testDerivedDisplayNameAsFallback() {
+        let infoPlistWithoutRequiredKeys = """
+        <plist version="1.0">
+        <dict>
+        </dict>
+        </plist>
+        """
+        
+        let infoPlistWithoutRequiredKeysData = Data(infoPlistWithoutRequiredKeys.utf8)
+        
+        XCTAssertEqual(
+            try DocumentationBundle.Info(
+                from: infoPlistWithoutRequiredKeysData,
+                bundleDiscoveryOptions: nil,
+                derivedDisplayName: "Derived Display Name"
+            ),
+            DocumentationBundle.Info(
+                displayName: "Derived Display Name",
+                identifier: "Derived Display Name",
+                version: nil
+            )
+        )
+    }
+    
+    func testDisplayNameAsIdentifierFallback() {
```

Comment:
Good tests.

One third case that could be worth covering is when the identifier is provided but not the display name.